extensions/amp-social-share/1.0/amp-social-share.js

```diff
@@ -238,8 +238,8 @@ AmpSocialShare['delegatesFocus'] = true;
 
 /** @override */
 AmpSocialShare['props'] = {
-  'tabIndex': {attr: 'tabindex'},
-  'type': {attr: 'type'},
+  'tabIndex': {attr: 'tabindex', media: true},
+  'type': {attr: 'type', media: true},
```

Comment:
I don't think `type` makes sense - too structural. And other props could really conflict with it. For instance, we allow a user to customize content of a social-share button, but we won't be able to reconcile it with the `type` based on media. I think CSS is a good tool for that which we immediately support. E.g. 

```
<amp-social-share type="1"></>
<amp-social-share type="2"></>

@media (...) {
  amp-social-share[type="1"] {
    display: none;
  }
}
```

Reply:
Good point, removed.